cli-scripts/deploy-prod.js

```diff
@@ -0,0 +1,54 @@
+const prompt = require("prompt");
+const simpleGit = require("simple-git");
+const colors = require("colors/safe");
+const promptRun = require("prompt-run");
+
+const git = simpleGit("/Users/treygranderson/Desktop/dev/spruce");
+
+prompt.start();
+
+git
+  .init()
+  .then(() => () => {})
+  .then(() => git.branchLocal())
+  .then(({ current }) => {
+    if (current !== "master") {
+      console.log("Current branch is", current);
+      console.log(
+        colors.red(
+          "Error: You must be on master branch to deploy to production"
+        )
+      );
+      prompt.stop();
+    }
+  });
+
+prompt.get(
+  {
+    properties: {
+      confirmDeploy: {
+        description: colors.magenta(
+          "Are you sure you want to deploy to production? [Y/N]"
+        ),
+      },
+    },
+  },
+  function(err, { confirmDeploy }) {
+    const confirmDeployLowerCased = confirmDeploy.toLowerCase();
+    const isConfirmed =
+      confirmDeployLowerCased === "yes" || confirmDeployLowerCased === "y";
+    if (isConfirmed) {
+      promptRun({
+        command: "npm run deploy-prod:do-not-use-directly",
```

Comment:
You should add this script to here instead `npm run build:prod && BUCKET=evergreen.spruce npm run deploy`  So people cant just run it directly.

Reply:
we want it to still be able to be used directly if need be, for example, doing an emergency deploy. this is not a firewall between deploying to prod deirectly, it's just a safety measure. if someone still does runs a command with `do-not-use-directly` then, well, they should just not be allowed to deploy to prod again lol

Reply:
eh, i actually agree wit you. if we need to deploy directly we can just copy the deploy command from the cli script

Reply:
but this is easier so i might leave it + there is very low rist